src/main/java/persistence/entity/persister/SimpleEntityPersister.java

```diff
@@ -28,6 +32,15 @@ public void insert(Object entity) {
         Object id = jdbcTemplate.executeInsert(dmlGenerator.generateInsertQuery(entity));
         Table table = Table.getInstance(entity.getClass());
         table.setIdValue(entity, id);
+
+        List<Object> relatedEntities = table.getRelationValues(entity);
+        relatedEntities.stream()
+            .flatMap(relatedEntity -> ((Collection<Object>) relatedEntity).stream())
+            .forEach(relatedEntity -> {
+                Object subId = jdbcTemplate.executeInsert(dmlGenerator.generateInsertQuery(relatedEntity, entity));
+                Table subTable = Table.getInstance(relatedEntity.getClass());
+                subTable.setIdValue(relatedEntity, subId);
+            });
```

Comment:
```suggestion
        Object id = jdbcTemplate.executeInsert(dmlGenerator.generateInsertQuery(entity));
        Table table = Table.getInstance(entity.getClass());
        table.setIdValue(entity, id);

        List<Object> relatedEntities = table.getRelationValues(entity);
        relatedEntities.stream()
            .flatMap(relatedEntity -> ((Collection<Object>) relatedEntity).stream())
            .forEach(relatedEntity -> {
                Object subId = jdbcTemplate.executeInsert(dmlGenerator.generateInsertQuery(relatedEntity, entity));
                Table subTable = Table.getInstance(relatedEntity.getClass());
                subTable.setIdValue(relatedEntity, subId);
            });
```

코드가 너무 방대해요. 동료가 본다면 쉽게 유지 보수 하기 힘들지 않을까요?
어떻게 하면 좀 더 가독성이 좋을까요?

Reply:
메서드로 분리해서 작업하도록 하겠습니다!